docs/pages/filterable-list-control-panels.md

```diff
@@ -9,22 +9,152 @@ description: The filter control panel allows users to whittle down number of
   parameters, housed in an expandable above the list.
 variation_groups:
   - variations:
-      - variation_code_snippet: ""
-        variation_description: >-
-          ### Collapsed
-
-
-          ![Filterable list collapsed](/design-system/images/uploads/filterable_list_control_panels_collapsed_071620.png)
-
-
-          ### Expanded
-
-
-          ![Filterable list expanded](/design-system/images/uploads/filterable_list_control_panels_expanded_071620.png)
+      - variation_code_snippet: >-
+          <div id="o-filterable-list-controls"
+          class="o-filterable-list-controls">
+              <div class="o-expandable
+                  o-expandable__padded
+                  o-expandable__background
+                  o-expandable__border">
+                  <button class="o-expandable_header o-expandable_target o-expandable_target__expanded" type="button">
+                      <span class="h4 o-expandable_header-left o-expandable_label">
+                      Filter posts
+                      </span>
+                      <span class="o-expandable_header-right o-expandable_link">
+                          <span class="o-expandable_cue o-expandable_cue-open">
+                              <span class="u-visually-hidden-on-mobile
+                                  ">
```

Comment:
TRIVIAL: would be nice to move this to the line before `<span class="u-visually-hidden-on-mobile">`